R/utils-get_code_dependency.R

```diff
@@ -106,6 +106,7 @@ extract_calls <- function(pd) {
   calls <- Filter(function(call) !(nrow(call) == 1 && call$token == "';'"), calls)
   calls <- Filter(Negate(is.null), calls)
   calls <- fix_shifted_comments(calls)
+  calls <- remove_custom_assign(calls)
```

Comment:
(optional)

Having second doubts about my suggestion, particularly using default arguments vs. being explicit,

Consider removing the default value and explicitly define them here.

```suggestion
  calls <- remove_custom_assign(calls, ":=")
```

or 

```suggestion
  calls <- remove_custom_assign(calls, c(":="))
```

Reply:
You guys are spending too much time on something that is not even a public part of the API :P